packages/atomic/src/components/search/atomic-generated-answer/copy-button.tsx

```diff
@@ -0,0 +1,25 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import CopyIcon from '../../../images/copy.svg';
+import {Button} from '../../common/button';
+
+interface CopyButtonProps {
+  title: string;
+  isCopied: boolean;
+  onClick: () => void;
+}
+
+export const CopyButton: FunctionalComponent<CopyButtonProps> = (props) => {
+  return (
+    <Button
+      title={props.title}
+      part="copy-button"
+      style={'text-neutral'}
+      class={` p-2 rounded-md ${props.isCopied ? 'copied' : ''}`}
+      onClick={props.onClick}
+    >
+      <div class="icon-container text-neutral-dark">
+        <atomic-icon class="w-5" icon={CopyIcon}></atomic-icon>
```

Comment:
I think we should expose a part to style the icon

```suggestion
        <atomic-icon part="copy-button-icon" class="w-5" icon={CopyIcon}></atomic-icon>
```

Reply:
Out of scope, but this makes me realize we're currently not exposing parts on the FeedbackButton component.

Reply:
@fbeaudoincoveo I will add it and i will also add button state on hover, currently it's always dark gray i will change it to green and red accordingly.